src/pusher/batchAuthorizer.js

```diff
@@ -11,15 +13,50 @@ const stopQueueing = () => {
   queueingInterval = null
 }
 
+
+const isDeviceProcessChannel = channelName => {
+  const uuidRegString = RegexUtils.UUIDV4_REGEX_STRING
+  const regex = new RegExp(`private-device_${uuidRegString}_process_${uuidRegString}(|_streaming_data)$`)
+  return regex.test(channelName)
+}
```

Comment:
This does not belong here but I failed at abstracting this. It belongs into `DeviceProcessSpecificPusherChannel` but that would introduce a circular dependency. 

The authorizer is required by the OspinPusherClient. The OspinPusherCLient is required by the channels, and the batchauthoizer would import a channel for the function `isDeviceProcessChannel`. 

I thought about not automatically passing the batchAuthorizer into the OspinPusherClient and instead exporting it from the module, so that the consumer has to pass it in, but that seems bad design, too, because it is unlike something else will be passed it.